lib/execution_engine2/sdk/EE2Runjob.py

```diff
@@ -691,32 +709,81 @@ def _get_run_job_params_from_existing_job(job: Job, user_id: str) -> Dict:
         # Then the next fields are job inputs top level requirements, app run parameters, and scheduler resource requirements
         return run_job_params
 
-    def run(
-        self, params=None, as_admin=False, concierge_params: Dict = None
-    ) -> Optional[str]:
+    def _handle_job_requirements(
+        self, params: List[Dict], concierge_params: Dict = None, as_admin: bool = False
+    ) -> None:
         """
+        Modifies params in place with job requirements
+
         :param params: SpecialRunJobParamsParams object (See spec file)
         :param params: RunJobParams object (See spec file)
```

Comment:
Duplicate conflicting definitions of `params`

Reply:
Yea I was wondering what that was, so I left it.. Deleting it now